Formula/arangodb.rb

```diff
@@ -28,8 +28,11 @@ class Arangodb < Formula
       :revision => "bbe29730e70dba609b57c469e8f863f032fabf3e"
   end
 
+  patch :DATA
```

Comment:
Please make sure the bug is reported upstream, and add a link in a comment above

Reply:
added link to upstream patch